src/parser/mod.rs

```diff
@@ -10289,8 +10289,21 @@ impl<'a> Parser<'a> {
 
         Ok(ExprWithAlias { expr, alias })
     }
+    /// return new expression with alias
```

Comment:
```suggestion
    /// Parses an expression with an optional alias

    /// Examples:

    /// ```sql
    /// SUM(price) AS total_price
    /// ```

    /// ```sql
    /// SUM(price)
    /// ```
```

Maybe something roughly like this? Mentioning that the alias is optional and then I'm thinking it might not be worth spelling out how to use the parser here (since the parser doc already covers it) given it would be more work to keep in sync